theme/src/components/header.js

```diff
@@ -42,11 +43,7 @@ function Header({isSearchEnabled}) {
             >
               <StyledOcticon icon={MarkGithubIcon} size="medium" />
             </Link>
-            <Box
-              as="h2"
-              id="site-heading"
-              sx={{fontSize: 'unset', fontWeight: 'unset', m: 0, display: 'flex', alignItems: 'center'}}
-            >
+            <Box sx={{fontSize: 'unset', fontWeight: 'unset', m: 0, display: 'flex', alignItems: 'center'}}>
```

Comment:
We can remove this Box now. I only added it because we wanted this text wrapped in an h2.